README.md

```diff
@@ -1,4 +1,4 @@
-# Solarizer
+# Solarizer [![Build Status](https://travis-ci.org/PaulDebevec/solarizer.svg?branch=master)](https://travis-ci.org/PaulDebevec/solarizer)
```

Comment:
Great addition to the readme! Looking forward to seeing how it looks on the readme